stdlib/REPL/src/LineEdit.jl

```diff
@@ -1259,21 +1259,21 @@ refresh_line(s, termbuf) = refresh_multi_line(termbuf, s)
 default_completion_cb(::IOBuffer) = []
 default_enter_cb(_) = true
 
-write_prompt(terminal, s::PromptState) = write_prompt(terminal, s.p)
+write_prompt(terminal, s::PromptState, hascolor::Bool) = write_prompt(terminal, s.p, hascolor)
```

Comment:
Maybe call this just `color`? It's typically only set if the terminal has color, but it could be called with anything.